lib/matplotlib/axes/_base.py

```diff
@@ -3242,9 +3242,11 @@ def ticklabel_format(self, *, axis='both', style='', scilimits=None,
         axis : {'x', 'y', 'both'}, default: 'both'
             The axis to configure.  Only major ticks are affected.
 
-        style : {'sci', 'scientific', 'plain'}
+        style : {'sci', 'scientific', 'plain', '', None}, default: None
             Whether to use scientific notation.
             The formatter default is to use scientific notation.
+            Sci is equivalent to scientific.
+            The '' option is included solely for backwards-compatibility.
```

Comment:
Apologies, I think I was unclear.  I meant to add a code comment where the dictionary is defined, so that future developers know that it’s there for a reason.  I don’t think it’s needed in the docstring.

Reply:
Done in the latest commit!